src/IO/WriteBufferFromHTTP.cpp

```diff
@@ -51,4 +53,21 @@ void WriteBufferFromHTTP::finalizeImpl()
     WriteBufferFromOStream::finalizeImpl();
 }
 
+std::unique_ptr<WriteBufferFromHTTP> BuilderWriteBufferFromHTTP::build()
```

Comment:
let's keep it consistent
```suggestion
std::unique_ptr<WriteBufferFromHTTP> BuilderWriteBufferFromHTTP::create()
```